### ISSUE TITLE
Is it possible to download the Signature Proof of Documnet (SPOD)?

### ISSUE THREAD

**Issue by adityaduggal** [python-fedex-devs/python-fedex#109]
Hi,

I would like to know if its possible to download the Signature proof of document (SPOD) by sending a SPOD request.

Ref: [Page No:86 from Documentation](https://images.fedex.com/templates/components/apps/wpor/secure/downloads/pdf/201507/FedEx_WebServices__DevelopersGuide_v2015.pdf)

**Comment by radzhome** [comment]
yes, if its possible via their API its possible with the wrapper.

**Comment by adityaduggal** [comment]
Hi,
I am sorry for asking a question on this issue which you have already closed. I would just like to know how could I send a SPOD request any example would be great for a person with limited knowledge about python.

Though I must say all other services are working flawlessly in your code.

**Comment by adityaduggal** [comment]
Hi @radzhome,

I have checked the wsdl files in the repository and found that there is indeed `GetTrackingDocumentsRequest` in `TrackServicev16.wsdl` 

Now I am kind of confused as to how can I make it to work in my app. I have tried the below but I am still unable to get the Signature Proof Downloaded from fedex.

```
from fedex.services.track_service import FedexTrackRequest
spod = FedexTrackRequest(credentials)

```
Now if I add the awb number and other details like below:

````
spod.SelectionDetails.PackageIdentifier.Value = awb_number 
````

And other details to the code I am unable to get the `GetTrackingDocumentsRequest` since there seems to be no option of doing that or atleast a newbie like me cannot seem to find that, I think it would be great if an example of that can be added to the documentation as well.
